Multisig/Data/Services/Notification Service/RegisterNotificationTokenRequest.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  RegisterNotificationTokenRequest.swift
+//  Multisig
+//
+//  Created by Moaaz on 8/5/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct RegisterNotificationTokenRequest: JSONRequest {
+    let uuid: UUID?
+    let safes: [String]
+    let cloudMessagingToken: String
+    let bundle: String
+    let version: String
+    let deviceType: String = "IOS"
+    let buildNumber: String
+    var httpMethod: String { return "POST" }
+    var urlPath: String { return "/api/v1/notifications/devices/" }
+
+    typealias ResponseType = Response
+
+    init(deviceID: UUID? = nil, safes: [Address], token: String, bundle: String, version: String, buildNumber: String) {
+        self.uuid = deviceID
+        self.safes = safes.map { $0.checksummed }
+        self.cloudMessagingToken = token
+        self.bundle = bundle
+        self.version = version
+        self.buildNumber = buildNumber
+    }
+
+    struct Response: Decodable {
+        let uuid: UUID
+        let safes: [String]
+        let cloudMessagingToken: String
+        let bundle: String
+        let version: String
+        let deviceType: String = "IOS"
+        let buildNumber: String
+    }
+}
+
+extension SafeTransactionService {
+
+    @discardableResult
+    func register(deviceID: UUID? = nil, safes: [Address], token: String, bundle: String, version: String, buildNumber: String) throws -> RegisterNotificationTokenRequest.Response {
```

Comment:
can you please add the default values for the bundle, version, and buildNumber to be taken from the App configuration?

Reply:
ok, no changes needed, saw your other file

Reply:
Is it right to do so in "Request" object?, I mean I that this should be done in upper level